pages/DSNP/DSNP-Message-Types.md

```diff
@@ -17,20 +17,21 @@
 * Message types are preset, but the DSNP Message contract will be upgradeable to allow more types.
 * Message type values outside the contract presets are invalid
 
-| name     | description | value |
-|-------    |-------------| ----|
-| Private | message and message type are private; message data is encrypted | 0 |
-| GraphChange | follow or unfollow an account | 1 |
-| Broadcast | a public post or reply | 2 |
-| Profile | a Profile change | 3 |
-| KeyList | KeyList rotation | 4 |
-| PrivateGraphKeylist | PrivateGraph keylist rotation | 5 |
-| EncryptionKeyList | Encryption keyList rotation | 6 |
-| Reaction | a public visual reply to a Broadcast | 7 |
-| PrivateGraphChange | an encrypted follow or unfollow | 8
-| Drop | a dead drop message | 9
-| EncryptedInbox | an encrypted direct message | 10
-| PrivateBroadcast | an encrypted broadcast | 11
+| name     | route | description | value |
+|-------   |-------------|-------------| ----|
+| Private | Personal Route ID | message and message type are private; message data is encrypted | 0 |
+| GraphChange | Personal Route ID | follow or unfollow an account | 1 |
```

Comment:
I'm not sure what the route should be for public graph changes (assuming of course that graph stays here). Thoughts?

Reply:
Why wouldn't it just be the social identity for the graph change?

Also, I think if we're going to keep "Private" then we don't need a PrivateGraphChange type.  But I can change that in my PR.

Reply:
Re: Private vs PrivateGraphChange. I think there is a lot of future discussion around this. Perhaps even a PR that merges them along with KeyList even

The idea behind the Personal Route ID is to note that it is not for routing to the public. These are not the messages you want if you want messages from a given person.